src/instructions/index.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { ExamStatus } from '../constants';
+import { StartExamInstructions } from './StartExamInstructions';
+import { SubmitExamInstructions } from './SubmitExamInstructions';
+import { SubmittedExamInstructions } from './SubmittedExamInstructions';
+import { startExam, continueExam, submitExam } from "../data";
+import { withExamStore } from "../hocs";
+import { isEmpty } from "../helpers";
+
+const Instructions = ({ attempt, examHasAttempt, children }) => {
+    switch (true) {
```

Comment:
we won't want to do this in another ticket but there's a few cases relating to an exam that is passed due.

Here's the existing logic for timed exam views which may be helpful: https://github.com/edx/edx-proctoring/blob/master/edx_proctoring/api.py#L1970

Reply:
Will add ticket for that